config/stage_test/local/kustomization.yaml

```diff
@@ -25,5 +25,5 @@ resources:
 
 images:
 - name: controller
-  newName: quay.io/cloudservices/floorist-operator
+  newName: quay.io/redhat-services-prod/insights-management-tenant/insights-floorist/floorist-operator
```

Comment:
Is there any chance to configure these images to be public, for anyone to test this locally?

Reply:
Ask Ernesto to make them public.

Reply:
EDIT: don't make this public, but request access via the link I DMed you